documentation/test-scenario/APITestErrorCode.md

```diff
@@ -0,0 +1,123 @@
+<!--
```

Comment:
It's a great starting point we have these error codes, can we add a section focus on each rule to describe how to fix? It's good to be more specific to guide service team to fix them, and it seems missing x-ms-secret or readonly are common issues spotting from the pilot

Reply:
Good point. Maybe I need to write a doc about validation logic. It will be covered in next PR.